src/pages/AddTransactionForm.jsx

```diff
@@ -41,7 +41,7 @@ export default function AddTransactionForm({fetchTransactions}) {
           addTransactionToDb(newTransaction) 
         }
         else if (today < transactionDate){
-          alert("Never know tomorrow ")
+          alert("That is in the future")
```

Comment:
Maybe we can update this to be: "The transaction date you entered is in the future. Please update the date to be today's date or before today."